kustomize/data-plane/deployDataPlane.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source common_tools.sh
+
+oc get project istio-system
+status=$?
+if [ $status -ne 0 ]; then
+  echo "The istio-system project does not exist, creating"
+  oc new-project istio-system
+fi
+
+echo "Deploying all resources"
+oc kustomize . | oc apply -f -
+
+status=$?
+if [ $status -ne 0 ]; then
+  echo "Some resources fail to deploy (concurrency issues), redeploying"
```

Comment:
Are you able to provide a list of the resources that fail to deploy? Ideally we should provide a permanent fix to it

Reply:
This was taken from [1] - but I can see now that this script has been removed, I will remove that condition and see if it actually fails now.

[1] https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/ee48af577a981f6fd33c983147d1fc69a6ac29cf/kustomize/startMinikubeDeployLocalDev.sh#L60

Reply:
Thanks 👍